.gitignore

```
@@ -1,4 +1,9 @@
 venv/
 .idea/
 
+lambda_function.zip
+climate_net_api/lambda_function.zip
+data_to_rds/lambda_function.zip
```

Instead of listing each `lambda_function.zip` file separately, we can simplify by only specifying `lambda_function.zip` once, which will apply to all directories.

```suggestion
lambda_function.zip
```